src/test/java/seedu/realodex/logic/parser/RealodexParserTest.java

```diff
@@ -93,6 +97,14 @@ public void parseCommand_filterByRemark() throws Exception {
         assertEquals(new FilterCommand(new RemarkContainsKeyphrasePredicate(keyphrase)), command);
     }
 
+    @Test
```

Comment:
Great test! Perhaps you can consider adding more tests where the tag set contains only buyer/only seller 🤔